odoo/addons/openupgrade_records/models/openupgrade_record.py

```diff
@@ -40,6 +40,7 @@ def field_dump(self):
             'module',
             'mode',
             'model',
+            'no_update',
```

Comment:
`attachment` ;-)

Reply:
hahaha, this is what happens when we mix late hours, topics, and so on, changing...